README.md

```diff
@@ -52,8 +56,8 @@ var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
 
 How would I go about accessing the string "pizza" in the above array?
 
-```js
-// write code here
+```
```

Comment:
Yes this correct, but I might instead make a local variable, using `var myString = `